common/util/actions.py

```diff
@@ -1,10 +1,13 @@
+from functools import wraps
+
 import sublime
+
 from ...core.settings import GitSavvySettings
 
 
 def destructive(description):
     def decorator(fn):
-
+        @wraps(fn)
```

Comment:
Note that it is important that decorators use `wraps` to retain the function signature.  Without it decorating a main "run" function with `GsWindowCommand` wouldn't work.